init/cli/stop.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cli
+
+import (
+	"os"
+
+	"github.com/palantir/pkg/cli"
+	"github.com/pkg/errors"
+
+	"github.com/palantir/go-java-launcher/init/lib"
+)
+
+func stopCommand() cli.Command {
+	return cli.Command{
+		Name: "stop",
+		Usage: `
+Stops the process the PID of which is written to var/run/service.pid. Returns 0 if the process is successfully stopped
+or is not running and the pidfile is removed and returns 1 otherwise.`,
```

Comment:
Documentation should also include how long this will wait for a process to exit before it is considered 'failed to stop'